### ISSUE TITLE
Server error when focus on PDF bundle

### ISSUE THREAD

**Issue by kashizui** [codalab/codalab#1397]
```
2015-11-12 11:41:25,721 ERROR Internal Server Error: /api/bundles/0xe2fa97ebd721418db5e589e0a32ff86f/
Traceback (most recent call last):
  File "/Users/sckoo/Developer/codalab/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 138, in get_response
    response = response.render()
  File "/Users/sckoo/Developer/codalab/venv/lib/python2.7/site-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/Users/sckoo/Developer/codalab/venv/lib/python2.7/site-packages/rest_framework/response.py", line 59, in rendered_content
    ret = renderer.render(self.data, media_type, context)
  File "/Users/sckoo/Developer/codalab/venv/lib/python2.7/site-packages/rest_framework/renderers.py", line 81, in render
    indent=indent, ensure_ascii=self.ensure_ascii)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 250, in dumps
    sort_keys=sort_keys, **kw).encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xe4 in position 10: invalid continuation byte
[12/Nov/2015 11:41:25] "GET /api/bundles/0xe2fa97ebd721418db5e589e0a32ff86f/?_=1447328479938 HTTP/1.1" 500 18966
```

Causes the bundle permissions to fail to load:
![screen shot 2015-11-12 at 3 42 41 am](https://cloud.githubusercontent.com/assets/614837/11117542/7efdf0f6-88ef-11e5-8cbe-500c66ccdaef.png)

**Comment by kashizui** [comment]
a similar problem occurs when doing `cl info -v` on a binary bundle

**Comment by kashizui** [comment]
Fixed by #1405
